manual/creating_a_user_application/index.md

```diff
@@ -7,11 +7,14 @@ toc: true
 toc_sticky: true
 ---
 
-Using the example of creating and displaying a canvas, the following shows how you can create a standalone user application.<br/>
-Two scenarios are presented:
+Using the example of creating and displaying a canvas, the following shows how you can
+create a standalone user application.
 
-- Quitting ROOT when closing the canvas
-- Returning to the ROOT prompt when closing the canvas
+Three scenarios are presented:
+
+- Generating the PDF output of the canvas and quitting.
```

Comment:
may we should link the scenarios with the headings

Reply:
done